my-app/public/index.html

```diff
@@ -21,6 +21,10 @@
   <link href='https://fonts.googleapis.com/css?family=Kaushan+Script' rel='stylesheet' type='text/css'>
   <link href='https://fonts.googleapis.com/css?family=Droid+Serif:400,700,400italic,700italic' rel='stylesheet' type='text/css'>
   <link href='https://fonts.googleapis.com/css?family=Roboto+Slab:400,100,300,700' rel='stylesheet' type='text/css'>
+  <!-- Latest compiled and minified CSS -->
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/latest/css/bootstrap.min.css">
```

Comment:
Think maybe the rendering issue on landing page is caused by these new bootstrap files? Especially when we already have a bootstrap.min.css imported.

Reply:
YESSSS You fixed it!!!!

Reply:
Awesome!